localstripe/resources.py

```diff
@@ -314,13 +314,7 @@ def last4(self):
 
     @property
     def iin(self):
-        if self.brand == 'Amex':
-            # Amex corportate test iin
-            if self._card_number == '371449635398431':
-                return '371449'
-            # Amex personal test iin
-            return '378282'
-        return None
+        return self._card_number[:6]
```

Comment:
since this is amex-specific, shouldn't you keep this under `if self.brand == 'Amex':`?

Reply:
Nope, this is all cards!